DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: tinyplot
 Type: Package
 Title: Lightweight Extension of the Base R Graphics System
-Version: 0.3.0
+Version: 0.3.0.1
```

Comment:
Nit: can we stick with the '.99' suffix convention for dev versions? Much as I'd like to increment with each PR, I don't think we'll be disciplined to stick to it and this approach avoids _that_ kind of ambiguity at least. (Following data.table's lead here...)

Reply:
do we need the date in DESCRIPTION? I don't think I include it in any of my packages. And that seems like an incrementation annoyance as well.